Header.py

```diff
@@ -105,19 +125,60 @@ def generate(self, project):
 
     header += comment[0] + '\n'
     header += comment[1] + " " + f[0] + " for " + project + " in " + f[1] + '\n'
-    header += comment[1] + '\n'
+    header += comment[1] + " \n"
     header += comment[1] + " Made by " + self.fullname + '\n'
     header += comment[1] + " Login   " + self.get_mail() + '\n'
-    header += comment[1] + '\n'
-    header += comment[1] + " Started on  " + self.get_date() + " " + self.fullname + '\n'
+    header += comment[1] + " \n"
+    header += comment[1] + " Started on  " + self.get_date()+ " " + self.fullname + '\n'
     header += comment[1] + " Last update " + self.get_date() + " " + self.fullname + '\n'
     header += comment[2] + '\n'
 
     return header
 
+  def is_already_here(self):
+    file_name = self.get_file_name();
+    region_c = self.view.find("\/\*\n\*\*[\s\S]+for[\s\S]+in[\s\S]+\n\*\* \n\*\* Made by [\s\S]+\n\*\* Login [\s\S]+<[\s\S]+_[\s\S]@epitech.net>\n\*\* \n\*\* Started on[\s\S]+\n\*\* Last update[\s\S]+?\n\*\/", 0)
+    region_make = self.view.find("##\n## Makefile for[\s\S]+in[\s\S]+\n## \n## Made by [\s\S]+\n## Login[\s\S]+<[\s\S]+_[\s\S]@epitech.net>\n## \n## Started on[\s\S]+\n## Last update [\s\S]+?\n##", 0)
+    if (file_name.endswith(".c") or file_name.endswith(".h")):
+      if (region_c == None or region_c == sublime.Region(-1, -1)) :
+        return 0
+      else:
+        return 1
+    else:
+      if (file_name == "Makefile"):
+        if (region_make == None or region_make == sublime.Region(-1, -1)) :
+          return 0
+        else:
+          return 1
+      else:
+        return 0
```

Comment:
Do you think you can use comments['current_language'] in the regex instead of hard coded makefile and c comment?